HappyAnding/HappyAnding/TextLiteral.swift

```diff
@@ -0,0 +1,212 @@
+    static let update: String = "업데이트"
+    static let share: String = "공유"
+    static let delete: String = "삭제"
+    static let withdrawnUser: String = "탈퇴한 사용자"
+    static let defaultUser: String = "user"
+    
+    // MARK: - ExploreShortcutView
+    static let exploreShortcutViewTitle: String = "단축어 둘러보기"
+    
+    // MARK: - RecentRegisteredView
+    static let recentRegisteredViewTitle: String = "최신 단축어"
+    
+    // MARK: - LovedShortcutView
+    static let lovedShortcutViewTitle: String = "사랑받는 단축어"
+    
+    // MARK: - DownloadRankView
+    static let downloadRankViewTitle: String = "다운로드 순위"
+    
+    // MARK: - CategoryView
+    static let categoryViewTitle: String = "카테고리 모아보기"
+    static let categoryViewUnfold: String = "펼치기"
+    static let categoryViewFold: String = "접기"
+    
+    // MARK: - MyShortcutCardListView
+    static let myShortcutCardListViewTitle: String = "내가 작성한 단축어"
+    
+    // MARK: - ValidationCheckTextField
+    static let validationCheckTextFieldInvalid: String = "단축어 링크가 아니에요"
+    static let validationCheckTextFieldExcess: String = "입력할 수 있는 문자 수를 초과했어요"
+    static let validationCheckTextFieldOption: String = "(선택입력)"
+    static let validationCheckTextFieldPrefix: String = "https://www.icloud.com/shortcuts/"
+    
+    // MARK: - UserCurationListView
+    static let userCurationListViewAdd: String = "추천 모음집 작성"
+    
+    // MARK: - NicknameTextField
+    static let nicknameTextFieldLength: String = "최대 8자까지 입력할 수 있어요"
+    static let nicknameTextFieldEmoticon: String = "사용할 수 없는 문자가 포함되어 있어요"
+    static let nicknameTextFieldSpace: String = "공백 없이 한글 ,숫자, 영문만 입력할 수 있어요"
+    static let nicknameTextFieldDuplicateTitle: String = "닉네임 중복 확인"
+    static let nicknameTextFieldDuplicateSuccessMessage: String = "사용할 수 있는 닉네임이에요"
+    static let nicknameTextFieldDuplicateFailMessage: String = "이미 사용 중인 닉네임이에요"
+    static let nicknameTextFieldDuplicateSuccessLabel: String = "확인"
+    static let nicknameTextFieldDuplicateFailLabel: String = "다시 입력하기"
+    static let nicknameTextFieldTitle: String = "닉네임 (최대 8글자)"
+    static let nicknameTextFieldDuplicateCheck: String = "중복확인"
+    
+    // MARK: - WriteShortcutTitleView
+    static let writeShortcutTitleViewNameTitle: String = "단축어 이름"
+    static let writeShortcutTitleViewNamePlaceholder: String = "단축어 이름을 입력하세요"
+    static let writeShortcutTitleViewLinkTitle: String = "단축어 링크"
+    static let writeShortcutTitleViewLinkPlaceholder: String = "단축어 링크를 입력하세요"
+    static let writeShortcutTitleViewOneLineTitle: String = "한 줄 설명"
+    static let writeShortcutTitleViewOneLinePlaceholder: String = "단축어의 핵심 기능을 입력하세요"
+    static let writeShortcutTitleViewMultiLineTitle: String = "상세 설명"
+    static let writeShortcutTitleViewMultiLinePlaceholder: String = "단축어 사용법, 필수적으로 요구되는 사항 등 단축어를 이용하기 위해 필요한 정보를 입력하세요"
+    static let writeShortcutTitleViewEdit: String = "단축어 편집"
+    static let writeShortcutTitleViewPost: String = "단축어 등록"
```

Comment:
```suggestion
    static let writeShortcutTitleViewPost: String = "단축어 작성"
```
이 부분이 바뀐 워딩에 들어있지 않았는데, 프로필 화면에도 '내가 작성한 단축어' 라고 표시가 되어 있고 추천 모음집도 작성으로 워딩이 통일되어 있어 단축어 등록 -> 단축어 작성으로 워딩을 변경하는 건 어떨까요?